SourceS/paths.cpp

```diff
@@ -82,4 +106,19 @@ void SetConfigPath(const char *path)
 	AddTrailingSlash(configPath);
 }
 
+void SetTtfPath(const char *path)
+{
+	if (ttfPath == NULL)
+		ttfPath = new std::string;
+	*ttfPath = path;
+	AddTrailingSlash(ttfPath);
+}
+
+void SetTtfName(const char *path)
+{
+	if (ttfName == NULL)
+		ttfName = new std::string;
+	*ttfName = path;
+}

```

Comment:
Why don't we need `AddTrailingSlash` in this one?

Reply:
No because this is a filename, not a directory path

Reply:
Whats the difference? Aren't they just both string that need to be null-terminated like any other string?

Can you elaborate further?

Reply:
It's not for null termination. The directory paths need the slash so the filename can be stuck on the end later. The file is the end and needs to not have a slash.

Reply:
> Whats the difference? Aren't they just both string that need to be null-terminated like any other string?
> 
> Can you elaborate further?

null != slash 🤷

Reply:
Oh wow, I completely misread the other method and thought we were adding null termination. My bad, nothing to see here.